lib/inputs/pingbacks.js

```diff
@@ -46,16 +49,22 @@ module.exports = class Pingbacks {
       }
     });
 
-    // ping urls, incrementing the counts per-host
+    // ping urls
     await Promise.all(pings);
+
+    // cleanup sockets (lambda executions seem to need this periodically)
+    this._agents.http.destroy();
+    this._agents.https.destroy();
```

Comment:
This is what _(I think!) actually fixed the issue.  The [Node 12 docs](https://nodejs.org/docs/latest-v12.x/api/http.html#http_agent_destroy) say this shouldn't be necessary, but this appears to do something in the Lambda execution.